identity-verification/module_1/1.0 SETUP - Gather images and setup S3 buckets for IDV modules.ipynb

```diff
@@ -54,28 +54,34 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 9,
    "id": "95ebfa80",
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "AWS Region: eu-west-2\n",
+      "AWS Bucket: ckp-packaging-us-east-1\n",
+      "Stored 'bucket_name' (str)\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/Users/ckp/code/work/github.com/aws-samples/amazon-rekognition-code-samples/.venv/lib/python3.11/site-packages/IPython/extensions/storemagic.py:229: UserWarning: using autorestore/bucket_name requires you to install the `pickleshare` library.\n",
```

Comment:
Are these local directory paths meant to be here?

Reply:
+1 - I guess you want the output to be wiped from source.

Reply:
Good spot. Thank you! I have removed the outputs